templates/paragraphs/paragraph--oe-contact.html.twig

```diff
@@ -0,0 +1,14 @@
+{#
+/**
+ * @file
+ * Theme override to display the Contact paragraph type.
+ *
+ * @see ./modules/contrib/paragraphs/templates/paragraph.html.twig
+ */
+#}
+{% if title is not empty %}
```

Comment:
Please consider to use
```
{% if not paragraph.field_oe_title.isEmpty() %}
  <h2 class="ecl-u-type-heading-2">
    {{ content.field_oe_title }}
  </h2>
{% endif %}
```
to avoid using oe_theme_preprocess_paragraph__oe_contact